neqo-transport/src/send_stream.rs

```diff
@@ -148,6 +148,7 @@ enum RangeState {
 struct RangeTracker {
     // offset, (len, RangeState). Use u64 for len because ranges can exceed 32bits.
     used: BTreeMap<u64, (u64, RangeState)>,
+    cached: Option<(u64, Option<u64>)>,
```

Comment:
```suggestion
    /// A cache for the last unmarked range, which we check an awful lot more than we update.
    last_unmarked: Option<(u64, Option<u64>)>,
```

Reply:
@jesup any reason not to rename this as suggested?